src/routes/PaymentResponsePage.tsx

```diff
@@ -172,6 +175,11 @@ export default function PaymentResponsePage() {
                 {t("errorButton.close")}
               </Button>
             </Box>
+            {outcome === ViewOutcomeEnum.SUCCESS && (
+              <Box sx={{ width: "100%" }}>
```

Comment:
Discuss this with the designer showing the draft you made.
Maybe he'll prefer a width inlined with the previous element (only on desktop viewport I guess).
![Schermata 2023-07-14 alle 12 59 26](https://github.com/pagopa/pagopa-checkout-fe/assets/1122389/2e1275b3-61e5-465f-97a9-b8ee4f28d1ff)